app/views/eventDetails/regTypes.html

```diff
@@ -280,8 +280,12 @@ <h4 translate>Payment</h4>
                 <label translate>Minimum Payment to Register</label>
                 <div class="input-group">
                   <span class="input-group-addon"
-                    ><i class="fa fa-usd"></i
-                  ></span>
+                    ><i class="fa"
```

Comment:
Is this still using font awesome? If the `fa` style is exactly what you want, you can keep it. Just seems a little odd to have font awesome html and then not use font awesome for the icon.

Reply:
I have tried just to remove it and it become italic. These fa class unfortunately exists also in many other places so maybe it worth refactor it in the separate pull request, please let me know what you think

Reply:
Nothing's forcing you to use the `<i>` tag. It's just the element font awesome decided to use for their icons.